release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
 
 [pr-milestone-query]: https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+milestone%3Av1.14
 [tide]: https://git.k8s.io/test-infra/prow/tide
 
-## Official Build
-* `./gcbmgr stage release-1.12 --official --build-at-head --nomock`
+## Official Stage and Release
+
+To initiate staging the build for the official release, the `--official` flag is used. For example:
+
+`./gcbmgr stage release-1.12 --official --build-at-head --nomock`
 
 In addition to `v1.12.n` this will also build and stage the subsequent patch's
 `beta.0`, in this example `v1.12.(n+1)-beta.0`. Similar to [creating a new
 branch](#branch-creation), the staging step will take about twice as long, the
 release step will also take a couple of minutes more.
 
-## Debs and RPMs
-These require an additional layer of build and publish, which is currently still done by a Googler.  After building the official release, but before release notification, contact [Kubernetes Build Admins][kubernetes-build-admins] indicating for which build (for example: 1.12.0) Debs and RPMs are required.
+When staging is done, as usual, use the command `./gcbmgr release` with the `--buildversion=` specified when `./gcbmgr stage` is done.
+
+An overview of what the [official release day](https://docs.google.com/document/d/1K0B91lgeEiJTbT602VloA5arb6AkaTif-MLryaHmlYc/edit?usp=sharing) might entail.
+
+### Debian and RPM Packaging
 
-These build relatively quickly and should be available ahead of sending the release notification, especially for the official release when the worldwide community will attempt to get the new artifacts.  Since they build from the release tag, the first release step below is a pre-requisite for initiating the package builds.
+Packaging the Official Release is by conducted by employees at Google. Once `./gcbmgr release --offical ...` has completed, **before sending out the email notification**, contact the [Kubernetes Build Admins][kubernetes-build-admins] to notify them than an official release for vX.Y is complete and the release is ready to be packaged.
 
-TODO: How do we automate this? These should not just be made but also validated prior to the release-notify phase. See issue: [kubernetes/release/#631](https://github.com/kubernetes/release/issues/631)
+Packaging is relatively quick and should be available ahead of sending the release notification email. It is preferable to have the DEB and RPM files ready as the community worldwide will attempt to download the official release. Since packaging uses the release tag, it is important to [validate the release process](#release-validation).
+
+**NB**: How do we automate this? These should not just be made but also validated prior to the release-notify phase. See issue: [kubernetes/release/#631](https://github.com/kubernetes/release/issues/631)
```

Comment:
This TODO/NB line can be removed.  We have an open issue and work underway to improve this aspect of tooling.

Reply:
Mentioned/logged in #696 .